Resources/Textures/SS220/Clothing/Mask/Blue-shield-mask.rsi/meta.json

```
@@ -0,0 +1,27 @@
+{
+    "version": 1,
+    "license": "EULA/CLA with a hosting restriction, full text: https://raw.githubusercontent.com/SerbiaStrong-220/space-station-14/master/CLA.txt",
+    "copyright": "based on SS13 https://github.com/ss220-space/Paradise/blob/f04540b212de5bf1919f7f1a52a7e20f0892a00e/icons/mob/clothing/neck.dmi , modify for SS14 SS220 by 363679417022676992 (Discord),",
```

```suggestion
    "copyright": "taken from Paradise SS220 at https://github.com/ss220-space/Paradise/blob/f04540b212de5bf1919f7f1a52a7e20f0892a00e/icons/mob/clothing/neck.dmi",
```
ты ведь сам сказал, что он не модифицировал текстурки, а просто перерисовал пиксель-в-пиксель?